src/components/Button/Button.tsx

```diff
@@ -0,0 +1,64 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link Button }
+ */
+export interface ButtonProps extends BaseButtonProps {
+  buttonName?: string;
+  buttonType?: React.ButtonHTMLAttributes<HTMLButtonElement>["type"];
+  children: React.ReactNode;
+  disabled?: boolean;
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+}
+
+/**
+ * The button element, that can be used to help users
+ * carry out an action.
+ */
+export const Button: React.FunctionComponent<ButtonProps> = (
+  props: ButtonProps
+): JSX.Element => {
+  const {
+    id,
+    className,
+    isStartButton,
+    buttonName,
+    buttonType,
+    children,
+    disabled,
+    onClick
+  } = props;
+  return (
+    <button
+      id={id}
+      className={classNames("govuk-button", className, {
+        "govuk-button--start": isStartButton
+      })}
+      name={buttonName}
+      type={buttonType}
+      disabled
+      aria-disabled={disabled}
+      data-module="govuk-button"
```

Comment:
I don't think this is necessary given we're not using the `govuk-frontend` JavaScript. Do make sure that we're implementing the behaviour that it's there to support, though (https://github.com/alphagov/govuk-frontend/blob/063cd8e2470b62b824c6e50ca66342ac7a95d2d8/src/govuk/components/button/button.js).